data/json/terrain.json

```diff
@@ -984,6 +1037,7 @@
         "type" : "terrain",
         "id" : "t_wall_wood_broken",
         "name": "broken wood wall",
+        "description": "A number of planks are missing, and the structure is beginning to sag and fall apart.  It's going to need quite a bit of work to repair this wall.",
```

Comment:
Just a note to you as an author:

Don't be afraid to leave out commas! English allows one to be _very_ judicial with comma use (just ask [The Constitution](http://www.nytimes.com/2007/12/16/opinion/16freedman.html)) but sentences generally read more forcefully and easily with as few as one needs. It's kind of the Hemingway approach.

> A number of planks are missing, and the structure is beginning to sag and fall apart

This is grammatically fine. However, the comma introduces a pause, and one that's perhaps not necessary. Here and elsewhere consider:

> A number of planks are missing and the structure is beginning to sag and fall apart